src/Core/NotificationCenter/Enums/Priority.cs

```diff
@@ -0,0 +1,18 @@
+﻿#nullable enable
+using System.ComponentModel.DataAnnotations;
+
+namespace Bit.Core.NotificationCenter.Enums;
+
+public enum Priority : byte
+{
+    [Display(Name = "Critical")]
+    Critical = 0,
```

Comment:
💭 Should these be in a different order to more easily rely on a "normal" priority being 0?

Reply:
Fixed in 006f1b4